src/scripts/eastern_kingdoms/searing_gorge/searing_gorge.cpp

```diff
@@ -63,13 +63,25 @@ enum DoriusStonetenderData
     QUEST_ID_SUNTARA_STONES         = 3367
 };
 
+uint32 GAMEOBJECT_ID_SINGED_LETTER = 175704;
+uint32 GAMEOBJECT_GUID_SINGED_LETTER = 10106;
```

Comment:
Make those part of the enum above, not global variables.

Reply:
I did this because `ObjectGuid` isn't able to work out which instance of the overloaded function to use if both params are enums. I can cast them both to uint32 in the function call like below to resolve this:

```
        if (auto pGo = m_creature->GetMap()->GetGameObject(ObjectGuid(HIGHGUID_GAMEOBJECT, static_cast<uint32>(GAMEOBJECT_ID_SINGED_LETTER), static_cast<uint32>(GAMEOBJECT_GUID_SINGED_LETTER))))
```

Is this better?